adapters/ix/ix.go

```diff
@@ -409,8 +409,10 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 				unmarshalExtErr := json.Unmarshal(bid.Ext, &bidExt)
 				if unmarshalExtErr == nil && bidExt.Prebid != nil && bidExt.Prebid.Video != nil {
 					bidExtVideo = &openrtb_ext.ExtBidPrebidVideo{
-						Duration:        bidExt.Prebid.Video.Duration,
-						PrimaryCategory: bidExt.Prebid.Video.PrimaryCategory,
+						Duration: bidExt.Prebid.Video.Duration,
+					}
+					if bid.Cat == nil || len(bid.Cat) == 0 {
```

Comment:
Nitpick: Go has graceful handling of `nil`. It's safe to just use `len(bid.Cat) == 0` since `len` will return 0 for `nil`.

Reply:
Good catch. Changed